test/e2e/k8s/kubernetes.go

```diff
@@ -411,6 +410,7 @@ func (k TestKubernetes) DeleteSecret(secret *v1.Secret) {
 
 // RunOperator runs an operator on a Kubernetes cluster
 func (k TestKubernetes) RunOperator(namespace string) chan struct{} {
+	k.NewNamespace(namespace)
```

Comment:
Isn't this required due to the changes introduced with [tconst.RunSaOperator](https://github.com/infinispan/infinispan-operator/blob/master/test/e2e/main_test.go#L89-L94) ?

If we're going to create the namespace here, we should remove the call from ^ if statement.

Reply:
Yeah, @ryanemerson . The problem looks like not with NewNamespace invoke, but seams to be with RunSaOperator defaults